rs-libc/src/c/strcpy.c

```diff
@@ -1,8 +1,14 @@
 #include <string.h>
 
+#ifdef _MSC_VER
+#define RESTRICT __restrict
+#else
+#define RESTRICT restrict
+#endif
```

Comment:
Instead of adding this block in every file, can you just do the equivalent of `-Drestrict=__restrict` on the command line?